goentri.com.supertape.json

```diff
@@ -0,0 +1,43 @@
+{
+  "providerId": "goentri.com",
+  "providerName": "Entri",
+  "serviceId": "supertape",
+  "serviceName": "Supertape",
+  "version": 1,
+  "logoUrl": "https://cdn.goentri.com/logo.svg",
+  "description": "Allows user to easily set up domain using Entri",
+  "syncPubKeyDomain": "goentri.com",
+  "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com,app.goentri.com",
+  "variableDescription": "subdomain is the subdomain user is creating and subdomainPointsTo is to where this subdomain is pointed, hostValidation is the host for the domain validation and txtData is the validation key",
+  "sharedProviderName": true,
+  "hostRequired": false,
+  "records": [
+    {
+      "type": "CNAME",
+      "host": "www",
+      "pointsTo": "@",
```

Comment:
Sure about it? CMAME makes DNS same as APEX, but APEX is a redirect. IPs will match, but redirecting Webserver won't have any entry for www host so will return 404.